src/datasource.ts

```diff
@@ -568,6 +551,20 @@ export class OpenSearchDatasource extends DataSourceWithBackend<OpenSearchQuery,
     }
 
     // Frontend flow
+    const targetsWithInterpolatedVariables = request.targets.map((target) => {
```

Comment:
Just added interpolating template variables because adhoc filters are later added for Lucene and PPL in the frontend flow, e.g.
`this.queryBuilder.buildPPLQuery(target, adhocFilters, queryString)`